modules/cnf-about-the-profile-creator-tool.adoc

```diff
@@ -6,11 +6,21 @@
 [id="cnf-about-the-profile-creator-tool_{context}"]
 = About the Performance Profile Creator
 
-The Performance Profile Creator (PPC) is a command-line tool, delivered with the Node Tuning Operator, used to create the performance profile.
-The tool consumes `must-gather` data from the cluster and several user-supplied profile arguments. The PPC generates a performance profile that is appropriate for your hardware and topology.
+The Performance Profile Creator (PPC) is a command-line tool, delivered with the Node Tuning Operator, which can help you to create a performance profile for your cluster.
 
-The tool is run by one of the following methods:
+PPC discovery mode::
+Use the PPC tool in discovery mode to inspect your cluster. The PPC tool learns about your cluster in discovery mode by using the output from your `must-gather` data. 
 
-* Invoking `podman`
+Running the PPC::
+Using this information, and further configuration options specified in PPC command arguments, the PPC generates a performance profile that is appropriate for your hardware, topology and use-case.
```

Comment:
The output is an almost-ready-to-use profile, it might still need some tweaks. Not sure if we want to mention that.

Reply:
Updated to say, "the PPC generates a proposed performance profile ". Maybe that cover it?

Reply:
Yeah, that sounds accurate enough.